@types/adminLesson.d.ts

```diff
@@ -0,0 +1,10 @@
+export type Lesson = {
+  id: string
+  title: string
+  description: string
+  order: number
+  challenges: Challenge[]
+  docUrl: string
+  chatUrl: string
+  __typename: string
+}
```

Comment:
Can't you use the type from `graphql/index` because this comes from graphql?

Reply:
Let me try to see if that can work